Drivers should not spew stuff like this to the console at boot.
Make this message go only to the log, as most drivers do.

--- tun.c.~1~	Tue Jul 17 11:15:22 2012
+++ tun.c	Thu Feb 21 14:15:37 2013
@@ -166,7 +166,7 @@
 
 int _init(void)
 {
-  cmn_err(CE_CONT, "Universal TUN/TAP device driver ver %s "
+  cmn_err(CE_CONT, "?Universal TUN/TAP device driver ver %s "
 		   "(C) 1999-2000 Maxim Krasnyansky\n", TUN_VER);
 
   DBG(CE_CONT,"tun: _init\n");
